README.md

```diff
@@ -152,13 +152,11 @@ An `add-asset` command will look something like this:
 
 ```
 # Comment
-add-asset localfolder/derp.png images/8.png
```

Comment:
Are the quotes required? If not, we should add better documentation here. It might be worth adding a section on quoted blocks and when they can be used

Reply:
Quotes aren't required unless the string contains dashes or spaces. Where would that section go?

Reply:
Right before the "Licensing" section would be good. We should leave the existing documentation as is, but specify something like "If you wish to use these characters, see the section on quoted strings below."

Reply:
Will do.